Cabal/doc/nix-local-build.rst

```diff
@@ -419,6 +419,28 @@ have to separate them with ``--``.
 
     $ cabal new-run target -- -a -bcd --argument
 
+'new-run' also supports running script files that use a certain format. With
+a script that looks like:
+
+::
+
+    #!/usr/bin/env cabal
+    {- cabal:
+    build-depends: base ^>= 4.11
+                , shelly ^>= 1.8.1
+    -}
+
+    main :: IO ()
+    main = do
+        ...
+
+It can either be executed like any other script, using ``cabal`` as an
+interpreter, or through this command:
+
+::
+
+    $ cabal new-run script.hs
```

Comment:
Can you please also add an example of how to pass command-line arguments to a script started with `new-run`? I expect something like `cabal new-run script.hs -- --foo bar`.

Reply:
That's exactly right, I thought it was regular enough with the rest of `new-run` that it didn't need to be called out explicitly, but you're right that it could use a mention.